paddle/fluid/operators/mkldnn/fc_mkldnn_op.cc

```diff
@@ -166,6 +169,20 @@ class FCMKLDNNHandler
       }
     }
 
+    if (ctx.HasAttr("fuse_residual_connection") &&
+        ctx.Attr<bool>("fuse_residual_connection")) {
+      // For Inner Product primitives, the destination always N * C
+      auto residual_data = ctx.Input<phi::DenseTensor>("ResidualData");
+      auto residual_data_md =
+          dnnl::memory::desc({MB, OC},
```

Comment:
Does `residual_data_md` should have determined shape? Since `binary_add` has enabled broadcast, maybe {1, 1} or {MB, 1} is also good?